src/lib/translations/fr.js

```diff
@@ -75,10 +75,30 @@ export default {
       },
       set: {
         frequency: "Fréquence",
+
```

Comment:
Wondering if we want to use this to go back to the package/activity wording

Reply:
To be honest my brain has hard time rewiring things... and indeed a language pack might help. But the code needs probably some renaming too.  Input, Topic & Compound (less PBF) are ok but the Set is really problematic for me and often lead to ambiguous code `setId` is this a function setting an id or the id of the Set... mixing it with `const setIds = new Set()` and it's game over.